pods-framework/pods
# Set up read-only fields

## Description

Sets up correctly functioning read-only fields.

## Related GitHub issue(s)

Fixes #6583.

## Testing instructions

1. Compare to comment https://github.com/pods-framework/pods/issues/6583#issuecomment-1216761494
2. Use read-only Kitchen Sink CPT to test that all fields have read-only versions available.
3. Compare to the non-read-only Kitchen Sink CPT to check that editable fields have not been affected. 

## Changelog text for these changes

Bug: Read-only fields are now uneditable.
Bug: Fixes an issue from wp-scripts that was causing jest to fail. 
Enhancement: New read-only version of the File field has been started, which can be built on to eventually rewrite the Marionette version of the File field. 

## PR checklist
